conf-sample.edn

```diff
@@ -18,10 +18,10 @@
              :format "%s/ep-resource-%s-api"
              :version "0-SNAPSHOT"
              :modules #{addresses
-                        assets
                         availabilities
```

Comment:
Hmm, this list should be discovered from the classpath the same way the Java generator does, no? I can't see us asking customers to maintain this list for graphql when they don't for Helix.

Reply:
This conf is only needed for `boot` . For the maven plugin there is [full discoverability](https://github.com/elasticpath/rest-resource-viz/blob/301f973183fbdfa83326c1ce7c16253bdb8269b5/src/task/rest_resources_viz/extractor.clj#L363) and no lists to maintain.

Reply:
Running from `boot` is more of a development workflow I would say...